Bottom: f7819246ba7d961d763b09069582b28052ada856
Top:    b38b8cfc05b832cb00b3591c1521509abcf18839
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-13 12:47:12 +0800

Refresh of autofs-5.0.7-add-enable-sloppy-mount-option-to-configure.patch

---

diff --git a/lib/master.c b/lib/master.c
index 7663d57..836c0fd 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -1321,7 +1321,7 @@ int master_show_mounts(struct master *master)
 		printf("\nMount point: %s\n", ap->path);
 
 		/* Report nulled nulled entries */
-		if (cache_lookup_distinct(master->nc, path)) {
+		if (cache_lookup_distinct(master->nc, ap->path)) {
 			printf("\nmap entry %s nulled by previous entry\n",
 			       ap->path);
 		}
